.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: scala
+dist: trusty
+sudo: required
+group: edge
+jdk:
+  - oraclejdk8
+env:
+  matrix:
+    - SCRIPT="runJob consumer-service/consumer-service-java-sbt test"
+    - SCRIPT="runJob cors/cors-java test"
+    - SCRIPT="runJob cors/cors-scala test"
+    - SCRIPT="runJob file-upload/file-upload-scala-sbt test"
+    - SCRIPT="runJob http-header-handling/http-header-handling-java-sbt test"
+    - SCRIPT="runJob mixed-persistence/mixed-persistence-java-sbt test"
+    - SCRIPT="runJob mixed-persistence/mixed-persistence-scala-sbt test"
```

Comment:
only add `sbt` recipes at the moment.

Reply:
Is there a barrier to running `mvn` ones, or just trying to limit scope?

Reply:
Oh, forgot to comment that: I only focused on `sbt` at the moment. I don't think there's any limitation to run `mvn` just wanted to timebox.